libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -435,6 +435,18 @@ class Strings {
         assertPrints(matchDetails(inputString, toFind, 10), "Searching for 'ever' in 'Never ever give up' starting at position 10: Not found")
     }
 
+    @Sample
+    fun find() {
+        val text = "k1o2t3l4i5n6"
+
+        val firstNumberInText = text.find { it.isDigit() }
```

Comment:
Please inline an expression that uses `find` with `assertPrints` (for example, like in the sample for `last`). A temporary variable doesn't add an additional context here and only distracts from grasping the semantics.

Reply:
Well, it adds some context, but it might be better to add an explicit comment like `// the index of the first digit, '1'` and `// the string does not contain uppercase letters`.